capstone-backend/app.js

```diff
@@ -471,7 +569,18 @@ async function getUserData(user){
   const skillsQuery = new Parse.Query(Skills);
   skillsQuery.equalTo("User1", user);
   const userSkills = await skillsQuery.find();
-  return({skills: userSkills, roles: user.get("roles")})
+
+  const Company = Parse.Object.extend("Company");
+  const compQuery = new Parse.Query(Company);
+  compQuery.equalTo("User1", user);
+  const userCompanies = await compQuery.find();
+
+  const Language = Parse.Object.extend("Language");
+  const langQuery = new Parse.Query(Language);
+  langQuery.equalTo("User1", user);
+  const userLanguages = await langQuery.find();
+
```

Comment:
This is a good example of a function we can "modularize". The main function `getUserData` can be broken into 
`getUserSkills`, `getUserCompanies`, and `getUserLanguages`.

The same goes for many of the functions in this file.